daemon/daemon.go

```diff
@@ -221,11 +225,24 @@ func (d *Daemon) addRoutes() {
 	d.router.NotFoundHandler = NotFound("not found")
 }
 
+var shutdownMsg = i18n.G("snapd triggered a reboot to boot into an up to date system -- temprorarily disable the reboot by running 'sudo shutdown -c'")
```

Comment:
Suggested phrasing:

`"reboot scheduled to update the system - temporarily disable with 'sudo shutdown -c'"`